README.md

```diff
@@ -93,16 +115,22 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
-
+```
+Calling a function means invoking a function. when invoking a function it gives the result of that function. referencing means to assign name to an object.
 ```
```

Comment:
Referencing a function means to return the function definition.

Try defining a function in the console and then type out its name without `()`. What value is returned?